Dockerfile

```diff
@@ -108,6 +110,15 @@ RUN pip install -r requirements.txt -r requirements-dev.txt
 # Jenkins runs the containers with the `-u 1000:1000` option, not as root.
 RUN chown -R +1000:+1000 /venv
 
+# spead2_net_raw is needed so that qualification tests can use ibverbs
+RUN SPEAD2_VERSION=$(grep ^spead2== requirements.txt | cut -d= -f3) && \
+    cd /tmp && \
+    wget https://github.com/ska-sa/spead2/releases/download/v$SPEAD2_VERSION/spead2-$SPEAD2_VERSION.tar.gz && \
+    tar -zxf "spead2-$SPEAD2_VERSION.tar.gz" && \
+    cd "spead2-$SPEAD2_VERSION/src" %&& \
```

Comment:
What is the `%` for? When I try it in bash I get an error about too many arguments.

Reply:
I think that's finger trouble. Good catch. I wonder why the thing built... 🤔